src/Sinch/Fax/Emails/Emails.cs

```diff
@@ -0,0 +1,308 @@
+
+        /// <summary>
+        ///     Delete an email and associated numbers to that email to disable that email from sending and receiving faxes.
+        /// </summary>
+        /// <param name="email"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A successful task if response is 204.</returns>
+        Task Delete(string email, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        ///     Set the numbers for an email.
+        /// </summary>
+        /// <param name="email"></param>
+        /// <param name="phoneNumbers"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        Task<EmailAddress> Update(string email, string[] phoneNumbers, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        ///     List configured numbers for an email
+        /// </summary>
+        /// <param name="email">The email you want to list numbers for.</param>
+        /// <param name="pageNumber">Optional. The page to fetch. If not specified, the first page will be returned.</param>
+        /// <param name="pageSize">Number of items to return on each page.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        Task<ListNumbersResponse> ListNumbers(string email, int? pageNumber = 1, int? pageSize = 20,
+            CancellationToken cancellationToken = default);
+
+        /// <summary>
+        ///     Auto List configured numbers for an email
+        /// </summary>
+        /// <param name="email">The email you want to list numbers for.</param>
+        /// <param name="pageSize">Number of items to return on each page.</param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="pageNumber">Optional. The page to fetch. If not specified, the first page will be returned.</param>
+        /// <returns></returns>
+        IAsyncEnumerable<ServicePhoneNumber> ListNumbersAuto(string email, int? pageNumber = 1, int? pageSize = 20,
+            CancellationToken cancellationToken = default);
+    }
+
+    internal class EmailsClient : ISinchFaxEmails
+    {
+        private readonly string _projectId;
+        private readonly Uri _apiBasePath;
+        private readonly IHttp _http;
+        private readonly ILoggerAdapter<ISinchFaxEmails>? _logger;
+        private readonly Uri _baseAddress;
+
+
+        internal EmailsClient(string projectId, Uri baseAddress, ILoggerAdapter<ISinchFaxEmails>? loggerAdapter,
+            IHttp httpClient)
+        {
+            _logger = loggerAdapter;
+            _http = httpClient;
+            _projectId = projectId;
+            _baseAddress = baseAddress;
+            _apiBasePath = new Uri(baseAddress, $"/v3/projects/{projectId}/emails");
+        }
+
+        /// <inheritdoc />
+        public Task<ListEmailsResponse<string>> ListForNumber(string serviceId, string phoneNumber, int? pageNumber,
```

Comment:
As this request uses the `/services` path, it would make more sense to implement it the `Services` class and call it from there as it is exposed twice in the specification

Reply:
Yeah, makes sense, I will move it to `Services` in next PR